ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -346,9 +346,25 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; optn= []
       ; help= "Install a TLS CA certificate, pool-wide."
       ; implementation= With_fd Cli_operations.pool_certificate_install
-      ; flags= []
+      ; flags= [Deprecated ["Use pool-install-ca-certificate"]]
       } )
   ; ( "pool-certificate-uninstall"
+    , {
+        reqd= ["name"]
+      ; optn= []
+      ; help= "Uninstall a pool-wide TLS CA certificate."
+      ; implementation= No_fd Cli_operations.pool_certificate_uninstall
+      ; flags= [Deprecated ["Use pool-uninstall-ca-certificate"]]
+      } )
+  ; ( "pool-install-ca-certificate"
+    , {
+        reqd= ["filename"]
+      ; optn= []
+      ; help= "Install a TLS CA certificate, pool-wide."
+      ; implementation= With_fd Cli_operations.pool_certificate_install
```

Comment:
I think you should rename the functions in `Cli_operations` as well. You could link both old and new to functions with the new name.